Packs/MyXSIAMPack_MP/ParsingRules/ParsingRules.yml

```diff
@@ -0,0 +1,73 @@
+canonical: {
+  "okta_saas": [ ],
```

Comment:
it's unclear if you meant `<vendor>_<product>` or `<target_dataset>`. it should be the first in parsing rules

Reply:
I see. I am thinking that for sources we don't have data to we will just not put them. So instead of what I have here we will have this:
```yml
canonical: {
  "okta_on_prem": [
    {...}
]
```
(without the saas)